src/main/java/com/limechain/network/protocol/grandpa/messages/consensus/GrandpaConsensusMessage.java

```diff
@@ -0,0 +1,16 @@
+package com.limechain.network.protocol.grandpa.messages.consensus;
+
+import com.limechain.chain.lightsyncstate.Authority;
+import lombok.Data;
+
+import java.math.BigInteger;
+import java.util.List;
+
+@Data
+public class GrandpaConsensusMessage {
+    private long bestFinalizedBlock;
```

Comment:
It would be more appropriate to rename the current ```lastFinalizedBlock``` field to ```forcedChangeBlockNumber ``` or ```governanceBlockNumber```. This suggestion is based on the polkadot spec definition of the ```m``` field in the gradnpa consensus message. Additionally, it would be better to change its type from long to BigInteger to ensure consistency with the type used for other block number fields.

Reply:
I also think we should rename this as it is explained differently in the [spec](https://spec.polkadot.network/sect-finality#sect-finality-forced-changes). Maybe something like `delayStartBlockNumber`.